tests/Database/DatabaseEloquentIntegrationTest.php

```diff
@@ -60,7 +61,9 @@ public function setUp()
 
         $this->schema()->create('photos', function ($table) {
             $table->increments('id');
-            $table->morphs('imageable');
+            $table->unsignedInteger("imageable_id")->nullable();
+            $table->string("imageable_type")->nullable();
+            $table->index(["imageable_id", "imageable_type"]);
```

Comment:
The `morphs` call had to be replaced here because of this change: https://github.com/laravel/framework/pull/8586/files

The test that was introduced there was never actually working correctly. The model wasn't actually being saved because it was missing required columns. By manually creating the morph columns and making them nullable like that pull request was expecting, the test now correctly tests what it was trying to test.

Sort of odd, but thought it would be worth explaining.